packages/studio-ui/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "useUnknownInCatchVariables": false,
+    "strictNullChecks": true,
```

Comment:
if this rule applies to all files and not only files that are edited can you revert it

Reply:
It applies to all files. I think it's a good idea to adopt it now, as it will help Typescript catch a lot more errors (I know it helped me catch many bugs in my code).

I understand this adds a lot of red everywhere in the codebase, but I think it's mandatory to activate this feature sooner than later